cmd/app.go

```diff
@@ -658,40 +673,45 @@ func templateInstall(ctx *cli.Context) error {
 		app.TargetNamespace = namespace
 	}
 
+	app.Wait = ctx.Bool("wait")
+	// 300 seconds is the default value that helm already uses
+	if timeout := ctx.Int64("timeout"); timeout != 300 {
```

Comment:
You shouldn't need to do the check, just use whatever timeout is passed in. If helm decides it wants to change its default to 350 this would need to be updated.